### DISCUSSION TITLE
Fabric Test network: update node version

### DISCUSSION THREAD

**Discussion by leonardotkimura** [hyperledger/fabric#2612] [General]
Hello,

Please, update the node version supported in peer chaincode. Currently it is still in 12.16, and I have some features available only in version 15.6 :'(

https://hub.docker.com/layers/hyperledger/fabric-nodeenv/2.3/images/sha256-e96a9e421159d2ae1153acae9fbb96e6008f6615b08631b5a91be4d8e9318bab?context=explore

**Comment by denyeart** [comment]
See the Node chaincode version support here:
https://github.com/hyperledger/fabric-chaincode-node/blob/main/COMPATIBILITY.md
